adbms/src/main/scala/de/up/hpi/informationsystems/adbms/protocols/RequestResponseProtocol.scala

```diff
@@ -4,14 +4,27 @@ import de.up.hpi.informationsystems.adbms.definition.Record
 
 object RequestResponseProtocol {
 
+  /** A Request to a Dactor for which you expect a Response.
+    *
+    * Each Dactor that can reply to queries subclasses these traits with their own message types so one can match on
+    * expected message types
+    */
```

Comment:
I'm missing the link to the usage pattern with `Dactor.askDactor()`

Reply:
Or you link it the other way around: Link from `Dactor.askDactor()` to the messaging protocol